src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -264,10 +270,101 @@ private int LineBreaksAfter(SyntaxToken currentToken, SyntaxToken nextToken)
             return 0;
         }
 
+        private static bool All<TNode>(SyntaxList<TNode> items, Func<TNode, bool> predicate)
+            where TNode : SyntaxNode
+        {
+            bool result = true;
+            for (int i = 0; i < items.Count; i++)
+            {
+                result &= predicate(items[i]);
+            }
+
+            return result;
+        }
+
+        private static bool IsAutoAccessorList(SyntaxToken token)
+        {
+            if (token.Parent is AccessorListSyntax accessorList)
+            {
+                return All(accessorList.Accessors, a => a.Body == null);
+            }
+
+            return false;
+        }
+
+        private static bool IsAutoAccessorList(SyntaxNode? node)
+        {
+            if (node != null && node.Parent != null && node.Parent is AccessorListSyntax accessorList)
+            {
+                return All(accessorList.Accessors, a => a.Body == null);
+            }
+
+            return false;
+        }
+
+        private static bool IsAccessorListFollowedByInitializer(SyntaxToken token)
+        {
+            if (token.Parent != null && token.Parent is AccessorListSyntax accessorList)
+            {
+                var declaration = accessorList.Parent;
+                if (declaration == null)
+                {
+                    return false;
+                }
+
+                SyntaxNode? lastChild = null;
+                foreach (var item in declaration.ChildNodes())
+                {
+                    lastChild = item;
+                }
```

Comment:
don't do this.  Just check if the parent if the accessorlist is a PropertyDeclaration and grab hte initializer off of that.

Reply:
How could I grab the initializer from the BasePropertyDeclarationSyntax type?

Reply:
If I can use `PropertyDeclarationSyntax`, it has an `Initializer` property. Not sure if there is a good reason to use `BasePropertyDeclarationSyntax` instead.

Reply:
Ok I now check for `PropertyDeclaration` and take the `Initializer` property. The `BasePropertyDeclaration` doesn't have the `Initializer` property.

Reply:
You'll need to handler indexers as well.  So wahtever you do that deals with that case is fine.  Also, make sure you test prop/indexer to validate it is hititng this case and handling it properly.

Reply:
There is already one test on indexers (which I previously fixed to adapt to my changes) and it is passing the tests.
You can also see the new test I added. It looks to me that they cover all the possibilities. If I miss something, please let me know.